tests/integration/product-type-export-encoding.spec.js

```diff
@@ -0,0 +1,151 @@
+import 'babel-polyfill'
+import test from 'tape'
+import { SphereClient } from 'sphere-node-sdk'
+import fs from 'fs'
+import { randomBytes } from 'crypto'
+import glob from 'glob'
+import tempfile from 'tempfile'
+import path from 'path'
+import iconv from 'iconv-lite'
+
+// eslint-disable-next-line max-len
```

Comment:
I am not sure the next line is more than 80 characters...

Also can we use NODE_PATH to reduce the relative path.

You can look at the in-house boilerplate to see how that goes.